netbox/virtualization/tables/clusters.py

```diff
@@ -75,7 +75,8 @@ class ClusterTable(TenancyColumnsMixin, ContactsColumnMixin, NetBoxTable):
     )
     site = tables.Column(
         verbose_name=_('Site'),
-        linkify=True
+        linkify=True,
+        accessor='_site'
     )
```

Comment:
We probably want to replace this with `scope_type` & `scope` columns like we did for PrefixTable.